File: innovation/innovation_app/lib/models/plan.dart
class Plan {
  final String id;
  final String title;
  final int price; // em reais (ex: 99)
  final List<String> features;

  const Plan({
    required this.id,
    required this.title,
    required this.price,
    required this.features,
  });
}

File: backend/src/domain/models/plan.py
from __future__ import annotations

from datetime import datetime

from sqlalchemy import DateTime, Integer, String, Text, text
from sqlalchemy.orm import Mapped, mapped_column

from infrastructure.database.sql.base import Base


class Plan(Base):
    __tablename__ = "plans"

    id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
    name: Mapped[str] = mapped_column(String(120), nullable=False)
    price: Mapped[str] = mapped_column(String(40), nullable=False)
    features: Mapped[str | None] = mapped_column(Text, nullable=True)

    created_at: Mapped[datetime] = mapped_column(
        DateTime(timezone=True), server_default=text("CURRENT_TIMESTAMP"), nullable=False
    )
